openapi/openapi.yaml

```diff
@@ -0,0 +1,274 @@
+              - 'Health'
+              - 'Housing'
+              - 'Humanities'
+              - 'Information and Statistics'
+              - 'Income Security and Social Services'
+              - 'Law, Justice and Legal Services'
+              - 'Natural Resources'
+              - 'Other'
+              - 'Opportunity Zone Benefits'
+              - 'Regional Development'
+              - 'Science and Technology and Other Research and Development'
+              - 'Transportation'
+              - 'Affordable Care Act'
+          code:
+            type: string
+            enum:
+              - 'RA'
+              - 'AG'
+              - 'AR'
+              - 'BC'
+              - 'CD'
+              - 'CP'
+              - 'DPR'
+              - 'ED'
+              - 'ELT'
+              - 'EN'
+              - 'ENV'
+              - 'FN'
+              - 'HL'
+              - 'HO'
+              - 'HU'
+              - 'IS'
+              - 'ISS'
+              - 'LJL'
+              - 'NR'
+              - 'O'
+              - 'OZ'
+              - 'RD'
+              - 'ST'
+              - 'T'
+              - 'ACA'
+      FundingActivity:
+        type: object
+        properties:
+          categories:
+            type: array
+            items:
+              $ref: '#/components/schemas/FundingActivityCategory'
+          explanation:
+            type: string
+      FundingInstrument:
+        type: object
+        properties:
+          name:
+            type: string
+            enum: [
+              'Cooperative Agreement',
+              'Grant',
+              'Procurement Contract',
+              'Other',
+            ]
+          code:
+            type: string
+            enum: [
+              'CA',
+              'G',
+              'PC',
+              'O',
+            ]
+      GrantorContact:
+        type: object
+        properties:
+          email:
+            $ref: '#/components/schemas/Email'
+          text:
+            type: string
+      Metadata:
+        type: object
+        properties:
+          version:
+            type: string
+      OpportunityCategory:
+        type: object
+        properties:
+          name:
+            type: string
+            enum:
+              - 'Discretionary'
+              - 'Mandatory'
+              - 'Continuation'
+              - 'Earmark'
+              - 'Other'
+          code:
+            type: string
+            enum:
+              - 'D'
+              - 'M'
+              - 'C'
+              - 'E'
+              - 'O'
+          explanation:
+            type: string
+      Opportunity:
+        type: object
+        properties:
+          id:
+            type: string
+          number:
+            type: string
+          title:
+            type: string
+          category:
+            $ref: '#/components/schemas/OpportunityCategory'
+      Grant:
+        type: object
+        properties:
+          grant_id:
```

Comment:
Should we create a new usdr-specific `grant_id` and rename this as `federal_grant_id` or something equivalent to that?

Reply:
My knee-jerk reaction is to say yes – I like the idea of our own independent identifier. My reservation is that I can't offhand think of an tangible reason for doing so, and given that the only planned consumer (GOST) only cares about the federally-issued identifier, it might make sense to wait until we something more concrete that can better-inform our needs around implementing a custom identifier. Normally I think I'd be fine just pulling the trigger, but given that federal grants are already inconsistent around canonical identifiers (Opportunity ID and Opportunity Number), I feel a little wary of wading into a similar predicament in our own, brand-new system.

Regarding this specifically:
> and rename this as `federal_grant_id` or something

Since the schema already provides the `Grant.opportunity.id` field, I think we could just use that instead of creating a new `federal_grant_id` field.

Reply:
makes sense - removed this field in favor of just keeping the grant-opportunity-id